src/main/java/com/ays/user/model/entity/UserEntity.java

```diff
@@ -108,8 +108,6 @@ public Map<String, Object> getClaims() {
         claims.put(AysTokenClaims.USERNAME.getValue(), this.username);
         claims.put(AysTokenClaims.USER_TYPE.getValue(), AysUserType.USER);
         claims.put(AysTokenClaims.ROLES.getValue(), List.of(this.role));
-        claims.put(AysTokenClaims.USER_FIRST_NAME.getValue(), this.firstName);
-        claims.put(AysTokenClaims.USER_LAST_NAME.getValue(), this.lastName);
```

Comment:
Eger kullanmayacaksak `AysTokenClaims `class'in `USER_FIRST_NAME `ve `USER_LAST_NAME` kaldirabilir miyiz?

Reply:
`AysTokenClaims` class'in icindeki `USER_FIRST_NAME` ve `USER_LAST_NAME` alanlarini kaldirdigimda AdminUserEntity bolumunde de degisiklik yapmak zorunda kaldim. Tekrar kontrol eder misin?

Reply:
Sadece `User `icin mi kullanacagiz yoksa `User` ve `Admin `icinde mi kullanacagiz? Ona bakmak lazim.